### PR TITLE
Change XPRO label + fixed misc layouts

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5815

*Description of changes:*
- cleaned up Tabs styling
- cleaned up ExistingRequestSearch layout
- changed XPRO to Extraprovincial in entity type label
- cleaned up Search layout
- changed border-radius to 4px consistently
- removed events/styles from active tab button

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/existing-request/existing-request-search.vue [hunk 1]
```diff
@@ -26,10 +28,11 @@
                       validate-on-blur />
       </v-col>
     </v-row>
-    <v-row align="center" class="ml-n9 mt-2" dense>
```
Comment (severinbeauvais) [inline client/src/components/existing-request/existing-request-search.vue:29]: Got rid of negative margin (that didn't work when screen was resized anyway), yay!

File: client/src/components/existing-request/existing-request-search.vue [hunk 2]
```diff
@@ -1,19 +1,21 @@
 <template>
-  <v-form v-model="isValid" lazy-validation @submit="handleSubmit()" class="mx-4 px-10 mb-9" ref="existing-nr-form">
```
Comment (severinbeauvais) [inline client/src/components/existing-request/existing-request-search.vue:2]: Got rid of wtf.

File: client/src/components/existing-request/existing-request-search.vue [hunk 3]
```diff
@@ -145,21 +149,17 @@ export default class ExistingRequestSearch extends Vue {
 }
 </script>
 
-<style lang="sass" scoped>
-.col
-  padding: 0
-  margin-top: 1.5rem
-
-.forgot-nr-link
```
Comment (severinbeauvais) [inline client/src/components/existing-request/existing-request-search.vue:153]: Got rid of obsolete styles. And changed to SCSS!

File: client/src/components/new-request/search.vue [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
       <v-col cols="6" class="pt-0 font-weight-bold h6"><span>I need a name to:</span></v-col>
       <v-col cols="6" class="pt-0">
         <span id="nr-required-activator"
               class="link-std d-flex justify-end"
               @click="activateNRRModal()">Check to see if you need to file a Name Request</span>
       </v-col>
-      <v-col cols="5" class="mt-n1">
+    </v-row>
+
+    <v-row class="mt-4" no-gutters>
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:12]: No Gutters gets rid of margins and padding for the row and cols.

So below (line 42), I added some inter-col padding back in.

File: client/src/components/tabs.vue [hunk 5]
```diff
@@ -1,37 +1,34 @@
 <template>
```
Comment (severinbeauvais) [inline client/src/components/tabs.vue:1]: I didn't change much here. Hide whitespace changes to see a cleaner diff!

File: client/src/sass/overrides.sass [hunk 6]
```diff
@@ -77,6 +77,7 @@
 
 .v-tab.active-tab:hover, .v-tab--active
   background-color: white !important
+  pointer-events: none
```
Comment (severinbeauvais) [inline client/src/sass/overrides.sass:80]: This is so the tab doesn't become clickable and go gray when you hover on it.

File: client/src/components/existing-request/existing-request-search.vue (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/components/existing-request/existing-request-search.vue:153]: nice!

File: client/src/sass/overrides.sass (hunk 6 above)
Comment (cameron-freshworks) [reply client/src/sass/overrides.sass:80]: Very cool

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Pull request merged**